### PR TITLE
Add Realtime, allowing realtime-js to initialise sockets from inside

### PR DESCRIPTION
**Pull request by kiwicopple**

### CODE REVIEW

File: packages/realtime-js/src/Realtime.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  }
+
+  /**
+   * takes in an object with keys
+   * - messageName
+   * - messgage 
+   * to:
+   * 1. add to this.listener
+   * 2. turn channel on
+   * @param {object} listener
```
Comment (kiwicopple) [inline packages/realtime-js/src/Realtime.js]: @dragarcia perhaps you can add 3 things to the JSdoc:

1. an` @example` 
2. change the @param to show the required items. i.e:
3. The description should be something like "Attaches a listener to the socket which gets fired on any change"

```
@param {object} listener Some description
@param {string} listener.channel Some description
@param {function} listener.callback A callback function to fire when the channel receives an change
```

File: packages/realtime-js/src/Realtime.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    this.channel.on(messageName, message)
+  }
+
+  /**
+   * Takes in messageName string to:
+   * 1. turn channel off
+   * 2. remove specific listener from this.listener
+   * @param {string} messageName 
+   */
+  removeListener(messageName){
```
Comment (kiwicopple) [inline packages/realtime-js/src/Realtime.js]: channel

File: packages/realtime-js/src/mapper.js [hunk 3]
```diff
@@ -80,6 +80,7 @@ export const convertCell = (type, stringValue) => {
       case 'cidr':
         return noop(stringValue)
       case 'date':
+        return noop(stringValue) // PostgREST uses string
```
Comment (kiwicopple) [inline packages/realtime-js/src/mapper.js]: Remove these too

**Comment by asterohype** [comment]
Test comment

**Pull request merged**